doc/hands_on/hands_on.md

```diff
@@ -0,0 +1,194 @@
+# Hands on S3-Virtual-Schema
+
+![Hands on S3-Virtual-Schema](./title.png)
+
+## Introduction
+
+In this guide we will walk through the setup of the Virtual Schema for document files stored on [AWS S3](https://aws.amazon.com/s3/).
+
+You need:
+
+* An Exasol database
+* An AWS account
+
+## What is This for?
+
+In this tutorial we will create a virtual schema for JSON-documents that we store in a S3 Bucket.
+If you've never heard of S3, that no problem. You can imagine it as a very basic file storage.
+You can store files and access them using a key.
+
+We are going to create a Virtual Schema for these files, so that you can access the JSON data just like a regular Exasol tabel.
+
+So you may ask yourself "Why not just create an Exasol table and import the data?".
+While in same cases this might even be the better solution, but there are also use cases where you still want to create a Virtual Schema:
```

Comment:
```suggestion
While in some cases this might even be the better solution, but there are also use cases where you still want to create a Virtual Schema:
```